src/DotLottie.tsx

```diff
@@ -0,0 +1,204 @@
+        UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          []
+        );
+      }
+    }, []);
+
+    const stopWithUIManager = useCallback(() => {
+      const command = NativeViewManager.Commands[COMMAND_STOP];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          []
+        );
+      }
+    }, []);
+
+    const setLoopWithUIManager = useCallback((value: boolean) => {
+      const command = NativeViewManager.Commands[COMMAND_SET_LOOP];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          [value]
+        );
+      }
+    }, []);
+
+    const setSpeedWithUIManager = useCallback((value: number) => {
+      const command = NativeViewManager.Commands[COMMAND_SET_SPEED];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          [value]
+        );
+      }
+    }, []);
+
+    const setPlayModeWithUIManager = useCallback((mode: Mode) => {
+      const command = NativeViewManager.Commands[COMMAND_SET_PLAY_MODE];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          [mode]
+        );
+      }
+    }, []);
+
+    const setFrameWithUIManager = useCallback((frame: number) => {
+      const command = NativeViewManager.Commands[COMMAND_SET_FRAME];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          [frame]
+        );
+      }
+    }, []);
+
+    const freezeWithUIManager = useCallback(() => {
+      const command = NativeViewManager.Commands[COMMAND_FREEZE];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          []
+        );
+      }
+    }, []);
+
+    const unfreezeWithUIManager = useCallback(() => {
+      const command = NativeViewManager.Commands[COMMAND_UNFREEZE];
+      if (command) {
+        return UIManager.dispatchViewManagerCommand(
+          findNodeHandle(nativeRef.current),
+          command,
+          []
+        );
+      }
+    }, []);
+
+    useImperativeHandle(ref, () => ({
+      play: playWithUIManager,
+      pause: pauseWithUIManager,
+      stop: stopWithUIManager,
+      setLoop: setLoopWithUIManager,
+      setSpeed: setSpeedWithUIManager,
+      setPlayMode: setPlayModeWithUIManager,
+      setFrame: setFrameWithUIManager,
+      freeze: freezeWithUIManager,
+      unfreeze: unfreezeWithUIManager,
+    }));
+
+    const parsedSource = parseSource(source);
+    return (
+      <DotlottieReactNativeView
+        ref={nativeRef}
+        loop={false}
+        autoplay={false}
```

Comment:
Is there a reason for hardcoding these props to false? Aren't they false by default?

Reply:
this stands as the default value.
but the props spread overrides it